backend/uclapi/workspaces/decorators.py

```diff
@@ -0,0 +1,17 @@
+from functools import wraps
+
+from .occupeye import OccupEyeApi
+
+
+def occupeye_api_request():
+    def occupeye_request(view_func):
+        @wraps(view_func)
+        def wrapped(request, *args, **kwargs):
+            api = OccupEyeApi()
+            # Make the API objects available to the function this
+            # decorator is attached to
+            kwargs['OccupEyeApi'] = api
+
+            return view_func(request, *args, **kwargs)
+        return wrapped
+    return occupeye_request

```

Comment:
Why is this created? Isn't it much simpler to create an object in the function?  Unless there is some performance constraints?

Reply:
Basically, I think I overestimated how much work would have to be done by a function to use the OccupEye API. Since I've moved most of the functionality into the `occupeye.py` file itself this can probably go.